src/commands/Minion/pc.ts

```diff
@@ -0,0 +1,90 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { BotCommand } from '../../lib/BotCommand';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { requiresMinion, minionNotBusy } from '../../lib/minions/decorators';
+import { Time } from 'oldschooljs/dist/constants';
+import { formatDuration } from '../../lib/util';
+import { rand } from '../../util';
+import { PestControlActivityTaskOptions } from '../../lib/types/minions';
+import { MinigameIDsEnum } from '../../lib/minions/data/minigames';
+import { Activity, Tasks } from '../../lib/constants';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+
+const landers = [
+	{ name: 'novice', points: 3, gamesRequired: 0, timeToFinish: Time.Minute * 4.5 },
+	{ name: 'intermediate', points: 4, gamesRequired: 100, timeToFinish: Time.Minute * 3.1 },
+	{ name: 'veteran', points: 5, gamesRequired: 200, timeToFinish: Time.Minute * 2 }
+];
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 3,
+			usage: '[quantity:int{1}]',
+			aliases: ['pestcontrol', 'pest-control'],
+			oneAtTime: true
+		});
+	}
+
+	@requiresMinion
+	@minionNotBusy
+	async run(msg: KlasaMessage, [quantity]: [null | number | string]) {
+		await msg.author.settings.sync(true);
+		if (msg.flagArgs.points) {
+			const points = msg.author.settings.get(UserSettings.CommendationPoints);
+
+			return msg.channel.send(`You have ${points} commendation points`);
+		}
+
+		const { maxTripLength, minionName } = msg.author;
+
+		const pcGames = msg.author.getMinigameScore(MinigameIDsEnum.PestControl);
+		const selectedLanders = landers.filter(({ gamesRequired }) => pcGames >= gamesRequired);
+
+		if (!selectedLanders.length) {
+			throw 'WTF?';
```

Comment:
not sure what this is doing, either give a meaningful message if this check is valid or dont have it at all